depends/packages/packages.mk

```diff
@@ -1,5 +1,5 @@
 zcash_packages := libsodium rustcxx utfcpp
```

Comment:
What's the intended distinction between `zcash_packages` and `packages`, btw? I don't see `zcash_packages` being used anywhere else; only to construct `packages` here.

Reply:
Packages that we introduce independently from upstream go into here. So `tl_expected` should have as well. But it doesn't matter too much.